Stripe/STPBankAccount.m

```diff
@@ -127,7 +124,7 @@ + (NSArray *)requiredFields {
              ];
 }
 
-+ (instancetype)decodedObjectFromAPIResponse:(NSDictionary *)response {
++ (nullable instancetype)decodedObjectFromAPIResponse:(nullable NSDictionary *)response {
```

Comment:
I went the other way in most of this, where I treated whatever was in `requiredFields` as currently correct, and tried to change the property declaration nullability to match. I might have missed a few, I can fix the header if needed. I think for now that is a better solution than changing `requiredFields`